frontend/javascripts/oxalis/view/components/setting_input_views.tsx

```diff
@@ -477,33 +493,87 @@ export class UserBoundingBoxInput extends React.PureComponent<UserBoundingBoxInp
       isVisible,
       onDelete,
       onExport,
+      onRegisterSegmentsForBB,
       isExportEnabled,
       onGoToBoundingBox,
       disabled,
       isLockedByOwner,
       isOwner,
     } = this.props;
     const upscaledColor = color.map((colorPart) => colorPart * 255) as any as Vector3;
-    const iconStyle = {
-      marginRight: 0,
+    const marginRightStyle = {
+      marginRight: 6,
+    };
+    const marginLeftStyle = {
       marginLeft: 6,
     };
-    const disabledIconStyle = { ...iconStyle, opacity: 0.5, cursor: "not-allowed" };
-    const exportIconStyle = isExportEnabled ? iconStyle : disabledIconStyle;
-    const exportButtonTooltip = isExportEnabled
-      ? "Export data from this bounding box."
-      : messages["data.bounding_box_export_not_supported"];
-    const exportColumn = isExportEnabled ? (
-      <Col span={2}>
-        <Tooltip title={exportButtonTooltip} placement="topRight">
-          <DownloadOutlined onClick={onExport} style={exportIconStyle} />
-        </Tooltip>
-      </Col>
-    ) : null;
+    const disabledIconStyle = { ...marginRightStyle, opacity: 0.5, cursor: "not-allowed" };
+    const exportIconStyle = isExportEnabled ? marginRightStyle : disabledIconStyle;
+    const exportButton = (
+      <>
+        <DownloadOutlined style={exportIconStyle} /> Export data
+      </>
+    );
     const editingDisallowedExplanation = messages["tracing.read_only_mode_notification"](
       isLockedByOwner,
       isOwner,
     );
+
+    const getContextMenu = () => {
+      const items: MenuProps["items"] = [
+        {
+          key: "registerSegments",
+          label: (
+            <>
+              <ScanOutlined style={marginRightStyle} /> {""}
+              Register all segments in this bounding box
+              <Tooltip title="Moves/registers all segments within this bbox into a new group">
```

Comment:
```suggestion
              <Tooltip title="Moves/registers all segments within this bounding box into a new segment group">
```

I'd avoid the term bbox when it's possible. the tab title is an exception because the space is scarce there.

Reply:
is it okay to name the newly created segment group with the registered segments `Segments for BBox <name>`, e.g. `Segments for BBox Bounding box 2`? it wouldnt feel good to have `Segments for Bounding Box Bounding box 2`, and I think this would occur often as the default name for bboxes is `Bounding box <no>`

Reply:
good question! how about `Segments for <name>`? I think, this should suffice in most cases. usually, the users use something like "bounding box" or "region" etc. and if not, they probably picked a very good name, anyway.

Reply:
okay! :)